ultranest/plot.py

```diff
@@ -77,8 +77,8 @@ class PredictionBand(object):
         band = PredictionBand(x)
         for c in chain:
             band.add(c[0] * x + c[1])
-        # add median line
-        band.line(color='k')
+        # add median line. As an option a matplotlib ax can be given.
+        band.line(color='k') # or band.line(color='k', ax = ax)
```

Comment:
Could you instead say below the code block:

    To plot onto a specific axis, use `band.line(..., ax=myaxis)`.

Reply:
Done